packages/destination-actions/src/destinations/linkedin-conversions/streamConversion/index.ts

```diff
@@ -18,7 +18,8 @@ const action: ActionDefinition<Settings, Payload, undefined, HookBundle> = {
       inputFields: {
         adAccountId: {
           label: 'Ad Account',
-          description: 'The ad account to use for the conversion event.',
+          description:
+            'The ad account to use when creating the conversion event. (When updating a conversion rule, changes to this field will be ignored. Ad Account IDs cannot be updated for a conversion rule.)',
```

Comment:
Can we rephrase this to clarify that the limitation is on LinkedIn's end, e.g.

```suggestion
            'The ad account to use when creating the conversion event. (When updating a conversion rule after initially creating it, changes to this field will be ignored. LinkedIn does not allow Ad Account IDs to be updated for a conversion rule.)',
```